models/lung_nodule_ct_detection/scripts/detection_inferer.py

```diff
@@ -25,14 +25,19 @@ class RetinaNetInferer(Inferer):
     Args:
         detector: the RetinaNetDetector that converts network output BxCxMxN or BxCxMxNxP
             map into boxes and classification scores.
+        force_inferer: whether to force using a SlidingWindowInferer to do the inference.
```

Comment:
Hi @Can-Zhao , may need your help to review this file. I'm not sure if the naming here is suitable. Thanks in advance.

Reply:
Looks good to me! Thank you!

Reply:
Yes, force_sliding_window sounds better